Src/library/net/Clib/network.c

```diff
@@ -29,12 +29,15 @@ indexing
 #define FD_SETSIZE 256
 #include <winsock2.h>
 #include <ws2tcpip.h>
+#include <Mswsock.h>
```

Comment:
When return value is FALSE, the GetLastError should be checked for WSA_IO_PENDING or ERROR_IO_PENDING that would mean the operation has been initialiated, but not yet completed. And if the Eiffel library should tell if the operation has been completed, the C code should check that as well (according to the documentation, TransmitFile puts one object to the signaled state on completion).